core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/BoxSize.java

```diff
@@ -25,8 +25,10 @@
   @GeneratedValue(strategy = GenerationType.AUTO)
   @Column(name = "boxSizeId")
   private long id = UNSAVED_ID;
-  private Integer rows;
-  private Integer columns;
+  @Column(name = "boxSizeRows")
+  private Integer boxSizeRows;
```

Comment:
`@Column` annotation isn't necessary if the Java field name matches the DB field name (though it also does no harm)